// slack/methods/users.js
'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _users = require('./users.getPresence');

var _users2 = _interopRequireDefault(_users);

var _usersProfile = require('./users.profile.get');

var _usersProfile2 = _interopRequireDefault(_usersProfile);

var _users3 = require('./users.identity');

var _users4 = _interopRequireDefault(_users3);

var _users5 = require('./users.info');

var _users6 = _interopRequireDefault(_users5);

var _users7 = require('./users.list');

var _users8 = _interopRequireDefault(_users7);

var _users9 = require('./users.setActive');

var _users10 = _interopRequireDefault(_users9);

var _users11 = require('./users.setPresence');

var _users12 = _interopRequireDefault(_users11);

var _usersProfile3 = require('./users.profile.set');

var _usersProfile4 = _interopRequireDefault(_usersProfile3);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

exports.default = {
  getPresence: _users2.default,
  identity: _users4.default,
  info: _users6.default,
  list: _users8.default,
  profile: {
    get: _usersProfile2.default,
    set: _usersProfile4.default
  },
  setActive: _users10.default,
  setPresence: _users12.default
};
module.exports = exports['default'];
